src/server/Backend.js

```diff
@@ -0,0 +1,31 @@
+class Backend {
```

Comment:
Please don't feel any obligation to follow the model from the Shadow Reader.  I wouldn't use it as an example here at all!  The Shadow Reader was originally created as a PWA where almost all the action happened on the client side.

In that case, Backend.js wasn't created to be a backend for the server, but to contain backend methods for the PWA.  When Andre and I wrote the Shadow Reader server, I simply reused that file because I needed some of its methods.

In our case, we can definitely have multiple JS files, but I wouldn't bother with this model. I'd just be consistent among all server files.

Reply:
Thanks Ben,

I'd like to add some kind of modularization on the backend, to decouple server.js from the actual data endpoint that we'll end up using. I'm using Backend.js here as a wrapper, with helper methods, but happy to discuss a different approach.